node-notes-cli/add.js

```diff
@@ -0,0 +1,11 @@
+const data = require('./data.json');
+const writeFileReuse = require('./writefile');
+const obj = data.notes;
+
+function create(string) {
+  obj[data.nextId] = string;
+  data.nextId = data.nextId + 1;
+  writeFileReuse(create);
```

Comment:
When you call `writeFileReuse` you should be passing it the `data` object, that is the data you want to write to the file. Not the function name. This applies everywhere you are calling `writeFileReuse`